src/citrine/ara/columns.py

```diff
@@ -31,15 +31,17 @@ def get_type(cls, data) -> Type[Serializable]:
         if "type" not in data:
             raise ValueError("Can only get types from dicts with a 'type' key")
         types: List[Type[Serializable]] = [
-            RealMeanColumn, IdentityColumn
+            IdentityColumn,
+            MeanColumn, StdDevColumn, QuantileColumn, OriginalUnitsColumn,
+            MostLikelyCategoryColumn, MostLikelyProbabilityColumn
```

Comment:
I don't think `MostLikelyProbabilityColumn` literally means what you want it to.  `HighestLikelihood`?

Reply:
Let's take a rep at those later.  cc: @gregor-robinson